apps/server/src/modules/board/uc/board.uc.ts

```diff
@@ -147,4 +143,24 @@ export class BoardUc {
 		await this.boardNodeService.updateVisibility(board, isVisible);
 		return board;
 	}
+
+	private async checkParentWritePermission(userId: EntityId, context: BoardExternalReference) {
```

Comment:
what happens if context = 'user'?
Will this be always a pass?

Reply:
Right, that should have been added. Did that.